patch/kconfig-inclusions

```diff
@@ -200,4 +200,15 @@ CONFIG_I2C_MUX_GPIO=m
 CONFIG_MTD_CMDLINE_PARTS=y
 CONFIG_MTD_OF_PARTS=y
 
+[nvidia-bluefield]
+CONFIG_MMC_DW=y
+CONFIG_MMC_DW_PLTFM=y
+CONFIG_MMC_DW_BLUEFIELD=y
+CONFIG_ARM64=y
+CONFIG_ACPI=y
+CONFIG_I2C=y
+CONFIG_I2C_SLAVE=y
+CONFIG_ARCH_SUPPORTS_ACPI=y
+CONFIG_EFIVAR_FS=y
```

Comment:
These lines are, I think, already set by other options or default values. Can they be removed?

Reply:
I think none of these are common or repeated

Reply:
They would generally apply to all ARM64 platforms. `CONFIG_ARM64`, for example, would also be needed by marvell-arm64.

Reply:
Okay, Makes sense to move CONFIG_ARM64 into the common arm64 section. 

But for others we are not certain that they can be safely moved to common section. If you think so, we can update.

Reply:
@saiarcot895 I removed all options enabled by default. Please check